common/src/main/java/com/wynntils/models/content/ContentModel.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.content;
+
+import com.wynntils.core.components.Model;
+import com.wynntils.core.text.StyledText;
+import com.wynntils.models.content.type.ContentDifficulty;
+import com.wynntils.models.content.type.ContentDistance;
+import com.wynntils.models.content.type.ContentInfo;
+import com.wynntils.models.content.type.ContentLength;
+import com.wynntils.models.content.type.ContentRequirements;
+import com.wynntils.models.content.type.ContentStatus;
+import com.wynntils.models.content.type.ContentTrackingState;
+import com.wynntils.models.content.type.ContentType;
+import com.wynntils.models.profession.type.ProfessionType;
+import com.wynntils.utils.mc.LoreUtils;
+import com.wynntils.utils.mc.StyledTextUtils;
+import com.wynntils.utils.type.Pair;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.world.item.ItemStack;
+
+public class ContentModel extends Model {
+    private static final Pattern LEVEL_REQ_PATTERN =
+            Pattern.compile("^§(.).À?§7(?: Recommended)? Combat Lv(?:\\. Min)?: (\\d+)$");
+    private static final Pattern PROFESSION_REQ_PATTERN = Pattern.compile("^§(.).À?§7 (\\w+)? Lv\\. Min: (\\d+)$");
+    private static final Pattern QUEST_REQ_PATTERN = Pattern.compile("^§(.).À?§7 Quest Req: (.+)$");
+    private static final Pattern DISTANCE_PATTERN = Pattern.compile("^   §7Distance: §.(\\w*)(?:§8 \\((.+)\\))?$");
+    private static final Pattern LENGTH_PATTERN = Pattern.compile("^   §7Length: (\\w*)(?:§8 \\((.+)\\))?$");
```

Comment:
Are you sure the spaces are constant (and does not change with longer lines)? Just asking, sometimes Wynn does align text with spaces

Reply:
It works for now. I have looked through all available content items. If it breaks in the future let's adjust it then. For now, I think it provides extra safety for not misinterpreting lore/description/next task.